modules/machine-api-overview.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='machine-api-overview-{context}']
+= Machine API overview
+
+On fully-managed {product-title} clusters, the Machine API performs all node
+management actions after the cluster installation finishes. Because of this
+system, {product-title} version 4.0 offers an elastic, dynamic provisioning
+method.
+
+The Machine API is a combination of primary resources that are based on the upstream
+link:https://github.com/kubernetes-sigs/cluster-api[Cluster API] project and
+custom {product-title} resources.
+
+The three primary resources are:
+
+`Machines`:: A fundamental unit that describes a node host. A `machine` has a
```

Comment:
> Looks like it should be just `Node`.

Then they're overloading :/.  Because from [here][1]:

> A node is a worker machine in Kubernetes, previously known as a **minion**.

Maybe they don't have a generic name for machine/hosts/whatever that is not overloaded?

[1]: https://kubernetes.io/docs/concepts/architecture/nodes/